### PR TITLE
Added a CHANGELOG entry for v1.8

### PR DESCRIPTION
**Pull request by philippjfr**

### CODE REVIEW

**Comment by jlstevens** [comment]
We need to mention ``config(style_17=True)`` and/or ``hv.extension(config=dict(style_17=True))`` in our release notes/CHANGELOG.

**Comment by philippjfr** [comment]
Yes, definitely should be listed in the backwards compatibility section.

**Comment by jlstevens** [comment]
I would also like to mention ``holoviews.rc`` with this in it:

```
import holoviews as hv
hv.extension.case_sensitive_completion=True
```

File: CHANGELOG.rst [hunk 1]
```diff
@@ -131,6 +137,11 @@ Changes affecting backwards compatibility:
 * Added ``opts`` method on all types eventually replacing the
   ``__call__`` syntax to set options (`#1589
   <https://github.com/ioam/holoviews/pull/1589>`_)
+* Styling changes for both matplotlib and bokeh which can be reverted
+  with ``hv.config(style_17=True)`` and/or
+  ``hv.extension(config=dict(style_17=True))`` (`#1518
```
Comment (jlstevens) [inline CHANGELOG.rst]: I would swap these around as the extension is probably the best way to do it.

**Comment by jlstevens** [comment]
@philippjfr If you are happy with my tweaks, please merge.

**Comment by philippjfr** [comment]
Looks good, merging.

**Pull request merged**